### ISSUE TITLE
add option to not error on "{filename} extra" output

### ISSUE THREAD

**Issue by stephen679** [vbatts/go-mtree#68]
From `mtree(8)`:
```
     -e    Do not complain about files that are in the file hierarchy, but not in the specification.
```

I think this would be useful for when we just want to know if the hierarchy derived from the specification is at least a subset of the actual file hierarchy it is being compared to.

**Comment by cyphar** [comment]
We currently don't detect this at all (we ignore extra files), but we can add this option after #48 is finished.

**Comment by stephen679** [comment]
I agree, this should be implemented after #48 is done. Although `TarCheck` does check for extra files

**Comment by cyphar** [comment]
Yes, sorry. I forgot that `Extra` was implemented (though I'll probably revamp it after #48, which IMO has a nicer interface for differences between manifests).

**Comment by vbatts** [comment]
This relates to #91
